.github/workflows/integration-tests.yaml

```diff
@@ -0,0 +1,34 @@
+name: Integration Tests
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+  pull_request:
+
+env:
+  CI: true
+
+jobs:
+  test:
+    if: ${{ github.repository == 'Ally-Guide/amplify-back-end' }}
```

Comment:
This will allow the workflow/job to be skipped for PRs from forked repositories (since they cannot be supplied with the Actions Secrets necessary to run the tests successfully).